### ISSUE TITLE
Import Shared Logging into First Product Codebase

### ISSUE THREAD

**Issue by jonyoder** [rstudio/platform-lib#6]
After #5, we need to Import the new logging code into one of our product codebases as the first adoption. Even if we don't use it fully immediately, we can ensure that it's easy to import the new library to avoid hiccups down the road.  This is being done with Connect.

- [x] Create integration code for the team
- [x] Work with team to get integrated

**Comment by jonyoder** [comment]
I've been working with importing the v0.1.0 release of `platform-lib` into Connect.

**Comment by bdeitte** [comment]
Connect team is going to do some manual testing to make sure this is all ok

**Comment by bdeitte** [comment]
This has been merged into the branch and the Connect team is testing this out
